docs/how-to/how-to-use-virtio-fs-nydus-with-kata.md

```diff
@@ -0,0 +1,17 @@
+# Kata Containers with virtio-fs-nydus
+
+## Introduction
+
+Research shows that time to take for pull operation accounts for 76% of container startup time but only 6.4% of that data is read. so if we can get data on demand (lazyload), it will speed up the container start. [Nydusd](https://github.com/dragonflyoss/image-service) is the project which build image with new format and can get data on demand when container start.
```

Comment:
A statement like this really needs more detail:

- Details of who carried out the research.
- A link to the research results (assuming they are public.

If you cannot do this, you could make the statement more generic maybe?

Reply:
done, have add link to the research